test/features/.nav

```diff
@@ -10,14 +10,16 @@
   - open-semi-colon-separated-file.feature
```

Comment:
@Stephen-Gates Will need to update this next time to reflect recent menu changes